tests/scripts/thread-cert/thread_cert.py

```diff
@@ -522,10 +524,36 @@ def _has_backbone_traffic(self):
         return False
 
     def _prepare_backbone_network(self):
-        network_name = config.BACKBONE_DOCKER_NETWORK_NAME
-        self.assure_run_ok(
-            f'docker network create --driver bridge --ipv6 --subnet {config.BACKBONE_PREFIX} -o "com.docker.network.bridge.name"="{network_name}" {network_name} || true',
-            shell=True)
+        """
```

Comment:
It looks like `_start_backbone_sniffer` is not updated to support different backbone link.

Reply:
Absolutely! Sniffer support is currently limited to backbone0.  We've got enhancements planned to enable multi-backbone support across all cases, which is tracked in https://github.com/openthread/openthread/issues/10551. I intentionally excluded this update from this PR to keep it small, and of course without breaking anything.